packages/web-components/src/components/expressive-modal/expressive-modal.scss

```diff
@@ -41,7 +41,7 @@
     grid-template-rows: auto 1fr auto;
   }
 
-  .#{$dds-prefix}-ce--modal__hedaer--with-body {
+  .#{$dds-prefix}-ce--modal__header--with-body {
```

Comment:
is this technically a breaking change for adopters using this class?

not sure if we need to keep both classes until the next major version bump

Reply:
Hm, I think we should be ok with this since this class is never applied manually right? Maybe this was a bug all along and this will fix?

Reply:
Actually @emyarod is right. We'd need to fix all corresponding element names as well

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/a5d805e58687616927b9a9322b08041081cf8a8f/packages/web-components/src/components/expressive-modal/expressive-modal.ts#L293

Reply:
https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/8757#discussion_r883961281